ui_framework/src/components/header/_header_logo.scss

```diff
@@ -0,0 +1,13 @@
+@include component('kuiHeaderLogo') {
+  position: relative;
+  height: $kuiHeaderChildSize;
+  padding: $kuiSize $kuiSizeXL;
+  display: inline-block;
+  vertical-align: middle;
+
+  @include child('icon') {
+    height: $kuiSizeXL !important;
```

Comment:
Why do we need `kui--medium` on the icon here? All it provides is size. This should be used instead of it, not on top of it?